### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by saschagrunert**
This also updates to the latest stable CNI releases.

```release-note
Update golang dependencies.
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,12 +3,11 @@ module github.com/cri-o/ocicni
...
-	github.com/sirupsen/logrus v1.4.2
-	github.com/vishvananda/netlink v1.1.0
-	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110 // indirect
+	github.com/containernetworking/cni v1.0.1
+	github.com/containernetworking/plugins v1.0.1
+	github.com/fsnotify/fsnotify v1.5.1
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.16.0
+	github.com/sirupsen/logrus v1.8.1
+	github.com/vishvananda/netlink v1.1.1-0.20210811191823-e1a867c6b452
```
Comment (aojea) [inline go.mod:12]: who is pulling this one?

Comment (aojea) [reply go.mod:12]: I see https://github.com/containernetworking/plugins/blob/189d0c06aa6da0c4f052d3831b091e6ea79e6675/go.mod#L22

**Comment by aojea** [comment]
/lgtm
ping me on the revendor to crio please

**Comment by haircommander** [comment]
@aojea would you mind retagging this?

**Comment by aojea** [comment]
> @aojea: changing LGTM is restricted to collaborators

:)

**Pull request merged**